packages/jest-environment/src/index.ts

```diff
@@ -42,6 +42,7 @@ export declare class JestEnvironment<Timer = unknown> {
   setup(): Promise<void>;
   teardown(): Promise<void>;
   handleTestEvent?: Circus.EventHandler;
+  exportConditions?: () => Array<string>;
```

Comment:
this could probably be a property, but making it into a function seems fine for a bit more flexibility